app/index.html

```diff
@@ -21,6 +21,9 @@
         <!-- build:js scripts/vendor/modernizr.js -->
         <script src="vendor/modernizr/modernizr.js"></script>
         <!-- endbuild -->
+        <!-- google api for geo complete-->
+        <script src="http://maps.googleapis.com/maps/api/js?sensor=false&amp;libraries=places"></script>
```

Comment:
Can you change this reference for now to be "//mapsgoogleapis.com.." instead of "http://maps.googleapis.com..."? That way it will work on heroku. In subsquent pull request, we can consider loading google maps asynchronously using require.js.